scan/lib/scan/error_handler.rb

```diff
@@ -24,7 +24,7 @@ def handle_build_error(output)
           print("https://stackoverflow.com/a/17031697/445598")
         when /Testing failed/
           UI.build_failure!("Error building the application - see the log above")
-        when /Executed/
+        when /Executed/, /Failing tests:/
```

Comment:
Can we add a comment inside of this `when` saying something like you mentioned in the PR description?
```
# "Executed" is not included in Xcode 10 parallel testing output - need to also look for "Failing tests:"
```

Or something like that so that others know why this is in here when coming through this code later 😊

Reply:
Done! Did it from the GH editor, so 🤞 I didn't cause a rubo👮‍♂️ warning :)

Reply:
That's a skill I'd like to have! I'm too good at making rubo warnings 😔

Reply:
Heh if you call rolling the dice skill! 😅

Reply:
There is nothing like a little successful yolo 👌